pyroclast/features/features.py

```diff
@@ -273,37 +273,38 @@ def plot_input_grads(data_dict,
     args_template = args
 
     models = {}
-    if not os.path.exists('mnist_normal'):
-        model_name = 'mnist_normal'
-        models[model_name] = learn(**args)
-
-    for lambd in [1e0, 1e1, 1e2]:
-        for alpha in [0., 1e0, 1e1, 1e2]:
+    for lambd in [0., 1e0, 1e1, 1e2]:
```

Comment:
Why not just use an itertools.product here? If we continue like this, every time we need to vary and stop varying another parameter, then the entire next section will have to change indentation.

Reply:
The whole point of adding Luigi is to deal with this problem. This code is moot as soon as Luigi is securely in place.